src/pdb/mod.rs

```diff
@@ -701,350 +603,70 @@ pub enum Row {
         /// Unknown field (alternating "2" and "3"?).
         unknown7: u16,
         /// International Standard Recording Code (ISRC), in mangled format.
+        #[br(offset = base_offset, parse_with = FilePtr16::parse)]
         isrc: DeviceSQLString,
```

Comment:
why did you opt for the `parse_with` approach instead?
```suggestion
        #[br(offset = base_offset)]
        isrc: FilePtr16<DeviceSQLString>,
```

Reply:
Because the actual offset is an implementation detail IMHO. When we add support for writing, I don't think we want to save the offsets in the struct, otherwise all later offsets of a row become wrong when an earlier string changes length.

Reply:
Mhmm, that makes sense, but since `FilePtr` does have BinWrite yet, it doesn't make much sense to discuss the ergonomics of a foreign, still unimplemented API.